### PR TITLE
Added invite event

### PR DESCRIPTION
**Pull request by TealNerd**

### CODE REVIEW

**Comment by rourke750** [comment]
Few things to comment on this will mention in a little.

**Comment by TealNerd** [comment]
kk, just want this for http://github.com/tealnerd/civguide

File: src/vg/civcraft/mc/namelayer/command/commands/InvitePlayer.java [hunk 1]
```diff
@@ -109,6 +109,7 @@ public boolean execute(CommandSender sender, String[] args) {
 		if(invitee.isOnline()){
 			//invitee is online make them a player
 			Player oInvitee = (Player) invitee;
+			Bukkit.getPluginManager().callEvent(new GroupInviteEvent(group, oInvitee, pType, p));
```
Comment (rourke750) [inline src/vg/civcraft/mc/namelayer/command/commands/InvitePlayer.java]: You need to check if the event was cancelled after it was fired and to escape if it was.

**Comment by TealNerd** [comment]
@rourke750 i guess this is kind of irrelevant now

**Pull request closed without merging**